animation_nodes/nodes/text/separate_text_object.py

```diff
@@ -235,6 +234,7 @@ def removeObject(object):
 def addObjectsToMainContainer(objects):
     mainContainer = getMainObjectContainer(bpy.context.scene)
     for object in objects:
+        bpy.context.collection.objects.unlink(object)
```

Comment:
This line seems wrong. Why should objects be unlinked from some collection in the `addObjectsToMainContainer` function?
Either the name should change or unlinking should be done in a separate function.
Also this line has the hidden assumption that the objects are within `bpy.context.collection`.

Reply:
Good point, I will put it into a separate function and will call it higher.